src/main/java/com/notably/logic/commands/DeleteCommand.java

```diff
@@ -15,36 +13,18 @@
 
     public static final String COMMAND_WORD = "delete";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
-
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    private final Path targetPath;
```

Comment:
Perhaps you would want to wrap in an `AbsolutePath` instead?
The `BlockManager` API now accepts `AbsolutePath`s only (#150)

Reply:
Good idea, will do that.

Reply:
After thinking about it i realise the user can supply both relative and absolute path thus i'll be changing it back to `Path` and changing it to `AbsolutePath` in the parse method.